From 33a474286b7e64429f73d9faf4f7e35e4b815bef Mon Sep 17 00:00:00 2001
From: Paul Holzinger <pholzing@redhat.com>
Date: Mon, 4 Jul 2022 18:09:30 +0200
Subject: [PATCH 3/3] pkg/machine: add missing build tags to tests

Machine can only run on amd64 and arm64 platforms so we need to make
sure the test are only run on those platforms. We do not have CI checks
for this but it fails in debian build infra since debian supports many
other architectures as well.

Signed-off-by: Paul Holzinger <pholzing@redhat.com>
---
 pkg/machine/config_test.go       | 3 +++
 pkg/machine/qemu/config_test.go  | 3 +++
 pkg/machine/qemu/machine_test.go | 3 +++
 3 files changed, 9 insertions(+)

--- a/pkg/machine/config_test.go
+++ b/pkg/machine/config_test.go
@@ -1,3 +1,6 @@
+//go:build amd64 || arm64
+// +build amd64 arm64
+
 package machine
 
 import (
--- a/pkg/machine/qemu/config_test.go
+++ b/pkg/machine/qemu/config_test.go
@@ -1,3 +1,6 @@
+//go:build (amd64 && !windows) || (arm64 && !windows)
+// +build amd64,!windows arm64,!windows
+
 package qemu
 
 import (
--- a/pkg/machine/qemu/machine_test.go
+++ b/pkg/machine/qemu/machine_test.go
@@ -1,3 +1,6 @@
+//go:build (amd64 && !windows) || (arm64 && !windows)
+// +build amd64,!windows arm64,!windows
+
 package qemu
 
 import (
